src/main/scala/Core/core.scala

```diff
@@ -144,6 +150,13 @@ class InstMemBusIO(implicit conf: FlexpretConfiguration) extends Bundle {
   val write = Input(Bool())
   val data_in = Input(UInt(32.W))
   val ready = Output(Bool()) // doesn't have priority
+
+  def driveDefaultsFlipped() = {
```

Comment:
Scaladoc?

Reply:
Very convenient to have a function for driving the IO to some default value (typically false.B and 0.U).  I normally add:
`driveDefaults()` and `driveDefaultsFlipped` to all my interfaces so that I don't have to manually drive the signals to default values

Reply:
I think it would be good to add this to the Scaladoc so we know what's going on

Reply:
@edwardcwang why are you pushing so much on Scaladoc. FlexPRET has not so much documentation itself. We would like to get this running ASAP, as I am leaving in a week.